### ISSUE TITLE
fix BC break on framework bundle twig keys

### ISSUE THREAD

**Issue by cordoval** [symfony/symfony#14488]
https://github.com/symfony/symfony/pull/14321

**Comment by xabbuh** [comment]
@cordoval Can you please explain the actual BC break? What did work before? What doesn't work anymore?

**Comment by cordoval** [comment]
@stof and @fabpot understood the problem. Let me try again to clarify but tbh my brain is wrappled into a lot of problem solving in another domain so not sure i can remember exactly.

Check this one https://github.com/braincrafted/bootstrap-bundle/pull/362/files i think if you install the bundle and use 2.7 you will see the exception and errors.

**Comment by xabbuh** [comment]
@cordoval Deprecating the `twig.form.resources` option is no BC break. Thus, I'm going to close this issue. Please feel free to reopen if you meant anything else and provide more information in that case.

**Comment by cordoval** [comment]
@nicolas-grekas is this fine? I thought it was a problem, i won't reopen i just hope it gets solved sometime.
